templates/distribution/scripts/pre-apply.sh.tpl

```diff
@@ -9,29 +9,79 @@ vendorPath="{{ .paths.vendorPath }}"
 
 {{- if index . "reducers" }}
 
+# ███████ ████████  █████  ██████  ████████ 
+# ██         ██    ██   ██ ██   ██    ██    
+# ███████    ██    ███████ ██████     ██    
+#      ██    ██    ██   ██ ██   ██    ██    
+# ███████    ██    ██   ██ ██   ██    ██    
+
+# Text generated with: https://www.patorjk.com/software/taag/#p=display&f=ANSI%20Regular&t=TRACING%20TYPE
+
+# ██       ██████   ██████   ██████  ██ ███    ██  ██████      ████████ ██    ██ ██████  ███████ 
+# ██      ██    ██ ██       ██       ██ ████   ██ ██              ██     ██  ██  ██   ██ ██      
+# ██      ██    ██ ██   ███ ██   ███ ██ ██ ██  ██ ██   ███        ██      ████   ██████  █████   
+# ██      ██    ██ ██    ██ ██    ██ ██ ██  ██ ██ ██    ██        ██       ██    ██      ██      
+# ███████  ██████   ██████   ██████  ██ ██   ████  ██████         ██       ██    ██      ███████ 
+                                           
 {{- if index .reducers "distributionModulesLoggingType" }}
 
 deleteOpensearch() {
   $kubectlbin delete --ignore-not-found --wait --timeout=180s ingress -n logging opensearch-dashboards
   $kubectlbin delete --ignore-not-found --wait --timeout=180s ingress -n pomerium opensearch-dashboards
   $kubectlbin delete --ignore-not-found --wait --timeout=180s ingress -n pomerium cerebro
-  $kustomizebin build $vendorPath/modules/logging/katalog/opensearch-dashboards | $kubectlbin delete --ignore-not-found --wait --timeout=180s -f -
-  $kustomizebin build $vendorPath/modules/logging/katalog/opensearch-triple | $kubectlbin delete --ignore-not-found --wait --timeout=180s -f -
-  $kustomizebin build $vendorPath/modules/logging/katalog/cerebro | $kubectlbin delete --ignore-not-found --wait --timeout=180s -f -
-  echo "Waiting 3 minutes"
-  sleep 180
-  $kubectlbin delete -l app.kubernetes.io/name=opensearch pvc -n logging --wait --timeout=180s
+
+  $kustomizebin build $vendorPath/modules/logging/katalog/opensearch-dashboards > delete-opensearch-cerebro.yaml
+  $kustomizebin build $vendorPath/modules/logging/katalog/opensearch-triple >> delete-opensearch-cerebro.yaml
```

Comment:
what if the user has selected the single version? will this work anyway or should we consider that?

Reply:
Since we are deleting, triple has only some environment variables and the replica set at 3, for a deletion it does not matter